docs/pages/database-access/guides/azure-sql-server-ad.mdx

```diff
@@ -0,0 +1,183 @@
+  Database Access for Azure SQL Server with Azure Active Directory authentication is
+  currently in Preview mode.
+</Admonition>
+
+This guide will help you to:
+
+- Install and configure Teleport.
+- Set up access to Azure SQL Server using Azure Active Directory authentication.
+- Connect to Azure SQL Server through Teleport.
+
+![Teleport Database Access Azure SQL Server Azure Active Directory](../../../img/database-access/guides/sqlserver/sql-aad.png)
+
+## Prerequisites
+
+- A SQL Server running on Azure.
+- Teleport Database Service running on an Azure virtual instance.
+
+(!docs/pages/includes/user-client-prereqs.mdx!)
+
+## Step 1/7. Set up the Teleport Auth and Proxy
+
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+
+## Step 2/7. Create a Teleport user
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+## Step 3/7. Enable SQL Server Azure Active Directory integration
+
+If you have it enabled, you can go to the next step.
+
+1. Go to the [Azure Portal](https://portal.azure.com/), select **Database
+   servers**, and select the database you wish to enable the Azure Active Directory
+   integration.
+2. Select **Azure Active Directory** in the left-hand column.
+3. Select **Set Admin**, and choose an account that will be added as an admin
+   login to SQL Server.
+
+![Azure SQL Server Azure Active Directory admin page](../../../img/database-access/guides/sqlserver/azure-set-ad-admin.png)
+
+## Step 4/7. Configure Azure Managed Identities
+
+The Teleport Database Service needs access tokens from Azure AD to authenticate with
+SQL Server databases.
+
+It uses the managed identities attached to its Virtual Machine to fetch the
+authentication token.
+
+To create a new user-assigned managed identity, go to the **Managed Identities**
+page in your [Azure Portal](https://portal.azure.com/#view/HubsExtension/BrowseResource/resourceType/Microsoft.ManagedIdentity%2FuserAssignedIdentities)
+and click on *Create*. Choose a name and resource group for it and create:
+
+![Azure Create user managed identity page](../../../img/database-access/guides/sqlserver/azure-user-managed-identity.png)
+
+Next, go to the **Teleport Database Service virtual machine instance**,
+**Identity** section, select **User assigned**, and add the identity we just
+created:
+
+![Azure Virtual machine user managed identities page](../../../img/database-access/guides/sqlserver/azure-attach-managed-identity-vm.png)
+
+## Step 5/7. Enable managed identities login on SQL Server
+
+Connect to your SQL Server using a pre-existent authentication method or the
+Activity Directory Admin set on
+[Step 3](#step-37-enable-sql-server-active-directory-integration).
+
+Create a [contained user](https://learn.microsoft.com/en-us/sql/relational-databases/security/contained-database-users-making-your-database-portable?view=sql-server-ver16)
+for your identities:
+```sql
+CREATE USER [sqlserver-identity] FROM EXTERNAL PROVIDER;
```

Comment:
@gabrielcorado What do you thinkg about putting also the following line here ? 
```
CREATE USER [sqlserver-identity] FOR LOGIN [sqlserver-identity]
```

Reply:
I've tested with a fresh SQL Server, and only having this query was enough to connect to the database. AFAIK the query you mentioned is used to create a contained user for a login, which seems to be possible only on Azure SQL Managed Instances:

> Because SQL Managed Instance supports Azure AD server principals (logins), using contained database users is not required.

In this case, we can create a login without having a contained user, meaning it can login into any database. I've opted for this single approach because it works in both instances. Do you think we should mention both methods?